### PR TITLE
feat: support show log level config by DDL 

### LINKED ISSUE #354: support dynamic modification of some configurations
### Describe your question

At present, many configuration items of openGemini are fixed when the process starts, and dynamic adjustment is not supported. In some scenarios, without restarting the process, some parameters need to be dynamically adjusted according to the actual situation.

#### syntax

```sql
set xxx="1G"
set xxx=true
set xxx=1000
```
#### parameters

I recommend the following parameters can be dynamically adjusted through DDL.

```text
data.write-cold-duration string
data.shard-mutable-size-limit string
data.node-mutable-size-limit  string
data.max-concurrent-compactions bool
data.compact-full-write-cold-duration string
```

#### syntax

```bash

```

#### parameters

### PR DESCRIPTION
**Pull request by xmh1011**
### What problem does this PR solve?

Issue Number: ref #354 
### What is changed and how it works?
Support `SHOW CONFIGS` command.
```
➜  openGemini git:(show-log) ts-cli
2023/10/08 17:45:22 proto: duplicate proto type registered: influxql.Measurements
2023/10/08 17:45:22 proto: duplicate proto type registered: influxql.Measurement
openGemini CLI 0.1.0 (rev-revision)
Please use `quit`, `exit` or `Ctrl-D` to exit this program.
> show configs
+-----------+----------+---------------+-------+
| component | instance | name          | value |
+-----------+----------+---------------+-------+
| sql       | :8086    | logging.level | info  |
+-----------+----------+---------------+-------+
4 columns, 1 rows in set

> 
```

### How Has This Been Tested?
Add unit test case in lib/config/config_test.go
Add integration test case in tests/server_test.go

First, you should start openGemini cluster. Then you can run as follows:
```shell
 URL=http://127.0.0.1:8086 go test -mod=mod -test.parallel 1 -timeout 10m ./tests -run  TestServer_ConfigCommand -v GOCACHE=off
```

```
➜  openGemini git:(show-log) ✗ URL=http://:8086 go test -mod=mod -test.parallel 1 -timeout 10m ./tests -run  TestServer_ConfigCommand -v GOCACHE=off
# github.com/openGemini/openGemini/tests.test
ld: warning: '/private/var/folders/w_/4l7k3gwn3g581ztmz26mzdc00000gn/T/go-link-3733628193/go.o' has malformed LC_DYSYMTAB, expected 137 undefined symbols to start at index 52935, found 157 undefined symbols starting at index 134
2023/10/08 17:46:46 proto: duplicate proto type registered: influxql.Measurements
2023/10/08 17:46:46 proto: duplicate proto type registered: influxql.Measurement
============= Running all tests for "tsi" index =============
=== RUN   TestServer_ConfigCommand
=== PAUSE TestServer_ConfigCommand
=== CONT  TestServer_ConfigCommand
=== RUN   TestServer_ConfigCommand/show_configs
=== RUN   TestServer_ConfigCommand/set_config
=== RUN   TestServer_ConfigCommand/show_configs#01
--- PASS: TestServer_ConfigCommand (0.53s)
    --- PASS: TestServer_ConfigCommand/show_configs (0.00s)
    --- PASS: TestServer_ConfigCommand/set_config (0.00s)
    --- PASS: TestServer_ConfigCommand/show_configs#01 (0.00s)
PASS

ok      github.com/openGemini/openGemini/tests  1.690s
```
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by xiangyu5632 — APPROVED** [review]

File: tests/server_suite.go [hunk 1]
```diff
@@ -1138,17 +1138,21 @@ func init() {
 			&Query{
 				name:    "show configs",
 				command: `SHOW CONFIGS`,
-				exp:     `{"results":[{"statement_id":0,"error":"impl me"}]}`,
+				exp:     `{"results":[{"statement_id":0,"series":[{"columns":["component","instance","name","value"],"values":[["sql","127.0.0.1:8086","logging.level","info"]]}]}]}`,
+				// when you run this integration test on macos, comment the previous line and use the next line
+				// exp:     `{"results":[{"statement_id":0,"series":[{"columns":["component","instance","name","value"],"values":[["sql",":8086","logging.level","info"]]}]}]}`,
```
Comment (shilinlee) [inline tests/server_suite.go:1143]:
```suggestion
```

File: tests/server_suite.go [hunk 2]
```diff
@@ -1138,17 +1138,21 @@ func init() {
...
-				exp:     `{"results":[{"statement_id":0,"error":"impl me"}]}`,
+				exp:     `{"results":[{"statement_id":0}]}`,
 			},
 			&Query{
 				name:    "show configs",
 				command: `SHOW CONFIGS`,
-				exp:     `{"results":[{"statement_id":0,"error":"impl me"}]}`,
+				exp:     `{"results":[{"statement_id":0,"series":[{"columns":["component","instance","name","value"],"values":[["sql","127.0.0.1:8086","logging.level","debug"]]}]}]}`,
+				// when you run this integration test on macos, comment the previous line and use the next line
+				// exp:     `{"results":[{"statement_id":0,"series":[{"columns":["component","instance","name","value"],"values":[["sql",":8086","logging.level","debug"]]}]}]}`,
```
Comment (shilinlee) [inline tests/server_suite.go:1155]:
```suggestion			
```

File: open_src/influx/coordinator/statement_executor.go [hunk 3]
```diff
@@ -2216,12 +2222,26 @@ func (e *StatementExecutor) executeDropStream(stmt *influxql.DropStreamsStatemen
 	return e.MetaClient.DropStream(stmt.Name)
 }
 
-func (e *StatementExecutor) executeShowConfigs(stmt *influxql.ShowConfigsStatement) error {
-	return fmt.Errorf("impl me")
+func (e *StatementExecutor) executeShowConfigs(stmt *influxql.ShowConfigsStatement) (models.Rows, error) {
+	row := &models.Row{Columns: []string{"component", "instance", "name", "value"}}
+	row.Values = append(row.Values, []interface{}{sqlConfig, app.BindAddress, loggingLevel, logger.Alevel})
```
Comment (shilinlee) [inline open_src/influx/coordinator/statement_executor.go:2227]: Do not add global variable `app.BindAddress`.

Suggest to add `hostname` to StatementExecutor structure.

The hostname should like:
`hostname := config.CombineDomain(s.config.HTTP.Domain, s.config.HTTP.BindAddress)`

**Review by shilinlee — CHANGES_REQUESTED** [review]

**Comment by shilinlee** [comment]
Rebase your codes from branch `main`.

**Comment by xmh1011** [comment]
> Rebase your codes from branch `main`.

ok

**Review by shilinlee — APPROVED** [review]

**Review by scuzyj — APPROVED** [review]

**Pull request merged**